templates/model/decode/decode_test.go

```diff
@@ -150,7 +151,7 @@ inputs:
   - name: 'foo'
```

Comment:
Do we need to add `builtin_vars` in the test case to verify the latest goldentest works?

Reply:
Yes, there's much more coming in future PRs.